### PR TITLE
[docs] Clarify storage support

### PR DESCRIPTION
**Pull request by alinaryan**
Adds a supported storage section to our docs that specifies that
only in-tree storage methods are supported in each cloud platform.

### CODE REVIEW

**Comment by Anandnatraj** [comment]
I would change the word "used" to "supported". Otherwise LGTM

**Comment by Anandnatraj** [comment]
LGTM

File: docs/wmco-prerequisites.md [hunk 1]
```diff
@@ -52,5 +52,8 @@ consistent across all supported cloud providers.
   
 * User-Provisioned Infrastructure is NOT supported.
 
+## Supported Storage
```
Comment (aravindhp) [inline docs/wmco-prerequisites.md:55]: This is not a pre-requisite but a limitation. I suggest adding a `Storage` sub-section under [limitations](https://github.com/openshift/windows-machine-config-operator#limitations)

**Review by aravindhp — CHANGES_REQUESTED** [review]
Thanks for adding this, @alinaryan

**Review by aravindhp — APPROVED** [review]

**Comment by mansikulkarni96** [comment]
Thanks for adding this @alinaryan 
/lgtm

**Review by mburke5678 — APPROVED** [review]
/lgtm

**Pull request merged**